teacher.rb

```diff
@@ -0,0 +1,12 @@
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization)
+    super(45, 'gala')
+    @specialization = specialization
+  end
```

Comment:
- [ ] in the initialization of this class you will need to include `name`, `age` and `parent_permission` in the arguments as well and super needs to refer to the arguments passed. Have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) again for more information.